watchtower.yml

```diff
@@ -0,0 +1,9 @@
+services:
+  watchtower:
+    image: containrrr/watchtower
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+      - /etc/localtime:/etc/localtime:ro
+    restart: always
```

Comment:
Tu valides qu'il se démarre 1 fois et reste démarré tout le temps ?
(histoire qu'il ne se redémarre pas en boucle)

Reply:
Yes it stays up and restarts on boot.